### DISCUSSION TITLE
modifica colore simboli sulla base della visualizzazione della scala

### DISCUSSION THREAD

**Discussion by SabrinaCasucci** [opendatasicilia/HfcQGIS-md#58] [Domande] [answered]
Ciao,
volevo cambiare il colore di visualizzazione di un simbolo;
per esempio alla scala tra 500 e 2000 lo vorrei vedere rosso e al di fuori di questa scala lo vorrei vedere giallo;
così ho scritto:
if(@map_scale > 500 and @map_scale<2000, e41a1c, ffff00)

C'è però un errore nella sintassi.

Grazie
Sabrina

**Comment by pigreco** [comment, marked as answer]
Ciao @SabrinaCasucci,
molto probabilmente l'errore è legato ai codici che utilizzi, forse sono incompleti,

prova cosi:

```
if(@map_scale > 500 and @map_scale < 2000 , '#e31a1c', '#e3ff1c')
```

oppure, scrivendo direttamnete il nome del colore:

```
if(@map_scale > 500 and @map_scale < 2000 , 'red', 'yellow')
```

**Reply by SabrinaCasucci** [reply]
Grazie mille!

**Reply by pigreco** [reply]
@SabrinaCasucci 
se è la risposta giusta per te, per favore clicca forte su `Mark as answer` :-)
